### PR TITLE
Odia | od_IN New Odia strings translated

### PR DESCRIPTION
**Pull request by soumendrak**
Around 70 new Odia language strings have been translated. Apologies for the auto new lines created.

### CODE REVIEW

File: locales/od_IN/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -234,6 +233,7 @@ msgstr "ଠିକଣା ପଟ୍ଟୀ:"
 #. as in multiple advertisements
 msgid "Ads"
 msgstr ""
+"ବିଜ୍ଞାପନଗୁଡ଼ିକ\n"
```
Comment (GioSensation) [inline locales/od_IN/LC_MESSAGES/duckduckgo.po:236]: Look out, here there is an extra `\n` that wasn't in the original string.

```suggestion
msgstr "ବିଜ୍ଞାପନଗୁଡ଼ିକ"
```

File: locales/od_IN/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -575,20 +576,26 @@ msgid ""
...
-"Browse as usual, and we’ll take care of the rest. Get bundled private "
-"search, tracker blocking, and site encryption, all in one download, for "
-"%smajor browsers%s."
+"Browse as usual, and we’ll take care of the rest. Get bundled private search,"
+" tracker blocking, and site encryption, all in one download, for %smajor "
+"browsers%s."
 msgstr ""
+"ନିୟମିତ ଭଳି ବ୍ରାଉଜ କରନ୍ତୁ, ଆମେ ବାକି ସବୁ ଦେଖିନବୁ । ଆମେ ଆମର ସନ୍ଧାନ ଇଞ୍ଜିନ, "
+"ଅନୁସରଣକାରୀ ଅବରୋଧକାରୀ ଏବଂ ଏନ୍କ୍ରିପସନ ଏନ୍ଫୋର୍ସର କୁ ଏକାଠି କରି ଗୋଟିଏ  %s%s "
+"ସ୍ମାର୍ଟ ବ୍ରାଉଜରରେ%s ରଖିଛୁ ।"
```
Comment (GioSensation) [inline locales/od_IN/LC_MESSAGES/duckduckgo.po:590]: Would you mind checking this again? It looks like some formatting got lost. In the original string, `%smajor browsers%s` is enclosed in `%s` tags. This must be kept in the translation. Thanks!

File: locales/od_IN/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -1329,6 +1352,12 @@ msgid ""
 "away, such that you remain anonymous. Learn more about how we designed this "
 "technology to protect your privacy %shere%s."
 msgstr ""
+"ଡିଜାଇନ୍ ଦ୍ୱାରା DuckDuckGo ବ୍ୟକ୍ତିଗତ ଅଟେ । ଯେତେବେଳେ ଆପଣ ଅବସ୍ଥାନ ସକ୍ଷମ କରନ୍ତି, "
+"ଏହା କେବଳ ଆପଣଙ୍କର ସ୍ଥାନୀୟ ଉପକରଣରେ ଗଚ୍ଛିତ ହୋଇଥାଏ । ଯେତେବେଳେ ଆପଣ ସନ୍ଧାନ କରନ୍ତି, "
+"ଆପଣଙ୍କର ଡିଭାଇସ୍ ତା’ପରେ ଏହାକୁ ଆମ ପାଖକୁ ପଠାଏ, ସେହି ସନ୍ଧାନ ପାଇଁ ଫଳାଫଳକୁ ଉନ୍ନତ "
+"କରିବା ପାଇଁ ଆମେ ଏହାକୁ ବ୍ୟବହାର କରୁ, ଏବଂ ତା'ପରେ ଆମେ ତୁରନ୍ତ ଏହାକୁ ଫିଙ୍ଗିଦେଉ, "
+"ଯେପରି ଆପଣ ଅଜ୍ଞାତ ହୋଇ ରହିଥାଆନ୍ତି । ଆପଣଙ୍କର ଗୋପନୀୟତା% shere%s ର ସୁରକ୍ଷା ପାଇଁ "
```
Comment (GioSensation) [inline locales/od_IN/LC_MESSAGES/duckduckgo.po:1359]: This also needs a check. The tag `%s` is supposed to surround the word `here`, like this: `%shere%s`. You can also translate the word `here` if you want. 😉

File: locales/od_IN/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -2401,7 +2453,7 @@ msgstr "ଅଧିକ ତଥ୍ୟ"
 
 msgctxt "Ads GDPR, internal use only. Do not change"
 msgid "More Info"
-msgstr ""
+msgstr "ଅଧିକ ସୂଚନା"
```
Comment (GioSensation) [inline locales/od_IN/LC_MESSAGES/duckduckgo.po:2456]: Sorry, this string is for internal use only and cannot be changed.

```suggestion
msgstr ""
```

File: locales/od_IN/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -3520,18 +3584,21 @@ msgid ""
 "Search privately with our app or extension, add private web search to your "
 "favorite browser, or search directly at %sduckduckgo.com%s."
 msgstr ""
+"ଆମର ଆପ୍ କିମ୍ବା ଏକ୍ସଟେନ୍ସନ୍ ସହିତ ବ୍ୟକ୍ତିଗତ ଭାବରେ ସନ୍ଧାନ କରନ୍ତୁ, ଆପଣଙ୍କର ପ୍ରିୟ "
+"ବ୍ରାଉଜରରେ ବ୍ୟକ୍ତିଗତ ୱେବ୍ ସନ୍ଧାନ ଯୋଡନ୍ତୁ, କିମ୍ବା% sduckduckgo.com%s ରେ "
```
Comment (GioSensation) [inline locales/od_IN/LC_MESSAGES/duckduckgo.po:3588]: Here as well, the formatting got lost.
```suggestion
"ବ୍ରାଉଜରରେ ବ୍ୟକ୍ତିଗତ ୱେବ୍ ସନ୍ଧାନ ଯୋଡନ୍ତୁ, କିମ୍ବା %sduckduckgo.com%s ରେ "
```

File: locales/od_IN/LC_MESSAGES/duckduckgo.po [hunk 6]
```diff
@@ -4508,14 +4596,16 @@ msgid ""
 "Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by "
 "Microsoft's ad network"
 msgstr ""
+"ବିଜ୍ଞାପନ ଦେଖିବା ହେଉଛି DuckDuckGo ଦ୍ୱାରା ଗୋପନୀୟତା ସୁରକ୍ଷିତ | ବିଜ୍ଞାପନ କ୍ଲିକ୍ "
+"Microsoftର ବିଜ୍ଞାପନ ନେଟୱାର୍କ ଦ୍ୱାରା ପରିଚାଳିତ |"
```
Comment (GioSensation) [inline locales/od_IN/LC_MESSAGES/duckduckgo.po:4600]: Sorry, this is for internal use only and cannot be changed.
```suggestion
msgstr ""
```

**Comment by GioSensation** [comment]
Thanks for your contribution, @soumendrak! I have left a few comments for you to look into. Let me know if something is not clear.

**Comment by soumendrak** [comment]
@GioSensation, Thanks for the input. Is there any tool I can validate before pushing again ? I modify with the online PO editor.

**Comment by GioSensation** [comment]
@soumendrak I use [Poedit](https://poedit.net). It highlights syntax errors and even formatting inconsistencies (like trailing whitespaces, inconsistent punctuation, etc.). We also have automated tests that check a bunch of different things (like branding, internal translations we don't want to change, and more), but that is on us. Hope it helps.

**Comment by GioSensation** [comment]
@soumendrak This PR hasn't been updated for quite a while. I will mark it stale and it will be closed in the next few days. If you need more time, please ping me here.

**Comment by soumendrak** [comment]
@GioSensation , Yes, I need more time please. I will do the change. Thank you.

**Comment by GioSensation** [comment]
Cool. Thanks for the update.

**Comment by GioSensation** [comment]
Hello @soumendrak. I'm pinging you again regarding this PR. I'm happy to leave it open if needed, but if you don't plan to work on this any longer, I'd ask you to please close it. Thanks!

**Comment by soumendrak** [comment]
I am on it; will update. Thanks @GioSensation

**Comment by soumendrak** [comment]
Will start a fresh PR. Closing this.

**Pull request closed without merging**